src/index.ts

```diff
@@ -29,13 +29,16 @@ const AnnotationServerStorage = (anno: any, settings: Settings) => {
             annotation.target.source,
             settings,
         );
+        // reassign ID to a URI on the annotation endpoint
```

Comment:
Let's make this comment clearer. Maybe something like this?

```suggestion
        // SAS requires a URI for annotation id; generate one by combining Annotorious generated ID with the annotation endpoint base URI
```

Reply:
Do we still need this code anymore? Can we use the remove and redisplay on the saved annotation?

I don't know how annotorious generates IDs, but I'm not sure we want to trust that they're going to be globally unique within the annotation server.